editoast/src/client/mod.rs

```diff
@@ -96,6 +97,8 @@ pub enum Commands {
     User(UserCommand),
     #[command(about, long_about = "Healthcheck")]
     Healthcheck(CoreArgs),
+    #[command(about, long_about = "Garbage collector")]
+    GarbageCollector,
```

Comment:
Maybe we coul rename this command.
`editoast gc` or `editoast garbage-collect`


```suggestion
    Gc,
```

Reply:
Done